### PR TITLE
fixed width of left sidebar

### PR DESCRIPTION
**Pull request by plata**
before

now

### CODE REVIEW

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/LeftButton.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public LeftButton(String name) {
         super(name);
         this.name = name;
         this.getStyleClass().add("leftButton");
-        this.setPrefWidth(Double.MAX_VALUE);
+        this.setMaxWidth(Double.MAX_VALUE);
```
Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/LeftButton.java]: Isn't this the default max width?
In any case I would prefer to see these definitions in the css files.

Comment (plata) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/LeftButton.java]: If it's not set, the buttons have different width. Is it possible to set it in CSS (how to say `Double.MAX_VALUE`)?

Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/LeftButton.java]: I believe you can set it to `Infinity` (see [here](https://stackoverflow.com/questions/26940341/how-to-set-fx-max-width-to-use-pref-size-in-javafx-css))

Comment (plata) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/LeftButton.java]: I have it already.

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/default/main.less [hunk 2]
```diff
@@ -101,16 +101,12 @@
...
-  -fx-min-width: 15em;
-  -fx-pref-width: 15em;
   -fx-padding: 1em;
 
   .leftPaneScrollbar {
     -fx-border-width: 0;
     -fx-background-color: @background-color;
-    -fx-min-width: 15em;
-    -fx-padding: 0.8em 1em 0.8em 1em;
+    -fx-padding: 0.8em 0 0.8em 0;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/default/main.less]: If you add this to the default theme you shouldn't need it in the other themes too, right?

Comment (plata) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/default/main.less]: Yes. But e.g. in the Unity theme it's already different. We have to do a complete cleanup sometime anyways.

Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/default/main.less]: Are you sure?
The unity theme doesn't define a `-fx-padding` for the `.leftPaneScrollbar`, so it should use the default one.

Comment (plata) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/default/main.less]: You're right. What I had in mind was the padding of `leftPane`.

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less [hunk 3]
```diff
@@ -304,14 +304,11 @@
 /*******************************************************/
 .leftPane {
     -fx-background-color: @main-color;
-    -fx-min-width: 15em;
-    -fx-pref-width: 15em;
     -fx-padding: 0 0 0.83em 0;
 
     .leftPaneScrollbar {
         -fx-border-width: 0;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: This is not needed, because the default theme already sets it

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/dark/main.less [hunk 4]
```diff
@@ -95,15 +95,11 @@
 /*******************************************************/
 .leftPane {
     -fx-background-color: @background-color;
-    -fx-min-width: 15em;
-    -fx-pref-width: 15em;
     -fx-padding: 1em;
 
     .leftPaneScrollbar {
         -fx-border-width: 0;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/dark/main.less]: This is not needed, because the default theme already sets it

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less [hunk 5]
```diff
@@ -357,6 +353,7 @@
         -fx-background-size: 2em;
         -fx-background-repeat: no-repeat;
         -fx-background-position: left;
+        -fx-max-width: Infinity;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less]: Shouldn't this be part of the default theme too?

Comment (plata) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less]: Sure. Somehow put it into the wrong file...

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less (hunk 3 above)
Comment (plata) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: Yes. But I don't want to change this now. As I said above, it's a general task to cleanup the CSS which does not belong to this PR.
